src/Search/SearchItem.tsx

```diff
@@ -59,14 +124,22 @@ const Title = styled.div`
 
 const Description = styled.div`
   font-size: 14px;
-  line-height: 22px;
   overflow: hidden;
   text-overflow: ellipsis;
 `;
 
-const Path = styled.div`
+const Place = styled.div`
   font-size: 12px;
-  line-height: 22px;
+  padding-top: 8px;
   overflow: hidden;
   text-overflow: ellipsis;
+  
+  &:first-child {
+    padding-top: 0;
+  }
+`;
+
+const Path = styled.div`
+  font-size: 12px;
+  line-height: 22px;
```

Comment:
I believe there are css variables from `font-size` and `line-height` properties. Try to use them